lint.swift

```diff
@@ -0,0 +1,547 @@
+        autoCorrectExamples: [
+            ["before": "let x = (viewModel?.profile?.imagePath)!\n", "after": "let x = viewModel!.profile!.imagePath\n"]
+        ]
+    )
+
+    // MARK: LateForceUnwrapping1
+    try Lint.checkFileContents(
+        checkInfo: "LateForceUnwrapping1: Don't use ? first to force unwrap later – directly unwrap within the parantheses.",
+        regex: [
+            "openingBrace": #"\("#,
+            "callPart1": #"[^\s\?\.]+"#,
+            "separator1": #"\?\."#,
+            "callPart2": #"[^\s\?\.]+"#,
+            "closingBraceUnwrap": #"\)!"#
+        ],
+        matchingExamples: ["call(x: (viewModel?.username)!)"],
+        nonMatchingExamples: ["call(x: (viewModel?.profile?.username)!)", "call(x: viewModel!.username)"],
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles],
+        autoCorrectReplacement: "$callPart1!.$callPart2",
+        autoCorrectExamples: [
+            ["before": "call(x: (viewModel?.username)!)", "after": "call(x: viewModel!.username)"]
+        ]
+    )
+
+    // MARK: MultilineIfStructure
+    try Lint.checkFileContents(
+        checkInfo: "MultilineIfStructure: Make sure multiline if conditions are all on their own lines, indented by four spaces.",
+        regex: #"^( *)if\s*(.*,) *\n((?:.+, *\n)*) *(.*\S)\s*\{\s*"#,
+        includeFilters: [swiftFiles],
+        excludeFilters: [testFiles],
+        autoCorrectReplacement: "$1if\n$1    $2\n$3$1    $4\n$1{\n$1    $5",
+        autoCorrectExamples: [
+            [
+                "before": """
+                    if let employee = UserDefaults.standard.currentEmployee(),
+                        let employeeName = employeeName,
+                        let employeeEmail = employeeEmail,
+                        let recipientEmail = recipientEmail {
+                            dataExporterHelper.exportData(
+                """,
+                "after": """
+                    if
+                        let employee = UserDefaults.standard.currentEmployee(),
+                        let employeeName = employeeName,
+                        let employeeEmail = employeeEmail,
+                        let recipientEmail = recipientEmail
+                    {
+                        dataExporterHelper.exportData(
+                """
+            ],
+            [
+                "before": """
+                    if let employee = UserDefaults.standard.currentEmployee(),
+                        let recipientEmail = recipientEmail
+                    {
+                        dataExporterHelper.exportData(
+                """,
+                "after": """
+                    if
+                        let employee = UserDefaults.standard.currentEmployee(),
+                        let recipientEmail = recipientEmail
+                    {
+                        dataExporterHelper.exportData(
+                """
+            ]
```

Comment:
Codacy found an issue: [Multi-line collection literals should have trailing commas.](https://app.codacy.com/gh/Flinesoft/HandySwift/pullRequest?prid=5730214)